x/avs/types/keys.go

```diff
@@ -15,13 +15,11 @@ const (
 
 	// RouterKey defines the module's message routing key
 	RouterKey = ModuleName
-
-	// MemStoreKey defines the in-memory store key
-	MemStoreKey   = "mem_avs"
+)
+const (
 	prefixAVSInfo = iota + 1
```

Comment:
This change is backwards incompatible. Should only be done at the time of upgrade. Alternatively replace this with `iota + 5` and keep a TODO for later.

See here https://github.com/ExocoreNetwork/exocore/pull/192/commits/1db91e8e83a8ddc9c94187a37d791e92d45be36b